debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/rest/DebeziumPostgresConnectorResource.java

```diff
@@ -53,6 +66,24 @@ public Connector getConnector() {
         return new PostgresConnector();
     }
 
+    @Override
+    public MetricsDescriptor getMetrics(String connectorName, MBeanServer mBeanServer) throws MalformedObjectNameException {
+        Map<String, String> connectorConfig = connectClusterState.connectorConfig(connectorName);
+        String serverName = connectorConfig.get("topic.prefix");
+        String tasksMax = connectorConfig.get("tasks.max");
+
+        ObjectName objectName = getObjectName("postgres", "streaming", serverName);
```

Comment:
We should use constants for those strings, basically based on the connector's metrics definition. If we don't have constants their they should be introduced. This can be filed as new Jira and done later for sure.